tests/testthat/test-build-reference-index.R

```diff
@@ -66,39 +66,46 @@ test_that("default reference includes all functions", {
 })
 
 test_that("errors well when a content entry is empty", {
-  meta <- yaml::yaml.load( "reference:\n- title: bla\n  contents:\n  - aname\n  - ")
-  pkg <- as_pkgdown(test_path("assets/reference"), override = meta)
+  meta <- yaml::yaml.load("reference:\n- title: bla\n  contents:\n  - aname\n  - ")
```

Comment:
no need to load, since `local_pkgdown_site()` can use meta as either character or list.

In another test file I saw

```r
  pkg <- local_pkgdown_site(test_path("assets/articles"), "
    template:
      bootstrap: 5
  ")
  ```
  
 which is more readable than the `\n`, although it might be awkward with the indentation.

Reply:
if not using the same "style" as above, I think I'd find seeing the list directly more readable than the  `\n`.

Reply:
Ah good catch; I forgot to update this older style.